src/Authoring/AuthoringConsumptionTest/packages.config

```diff
@@ -1,5 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.googletest.v140.windesktop.msvcstl.static.rt-dyn" version="1.8.1.3" targetFramework="native" />
+  <package id="Microsoft.VCRTForwarders.140" version="1.0.7" targetFramework="native" />
   <package id="Microsoft.Windows.CppWinRT" version="2.0.201113.7" targetFramework="native" />
+  <package id="Microsoft.WinUI" version="3.0.0-preview2.200713.0" targetFramework="native" />

```

Comment:
should keep in sync with MicrosoftWinUIVersion (3.0.0-preview3.201113.0)
could automate by generating packages.config, or leave comments in both places

Reply:
Updated version.  Turns out there is a couple other places using preview 2 still which I will look at updating all in another PR.  Might be a bit harder to automate given the vcxproj need to be updated, but will take a look.

Reply:
another technique would be to assert the WinUI version in the root directory.build.props/targets  - doesn't auto-config everything, but achieves the same end by breaking the build